e2e/examples/actions/redirect.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { test, expect } from '@playwright/test'
+
+test('With JS enabled', async ({ page }) => {
+  await page.goto('/examples/actions/redirect')
+
+  // Render
+  const firstNameInput = page.locator('input[name="firstName"]').first()
```

Comment:
Do we need `first()` on everything?

Reply:
Yup, unfortunately. Our `SidebarLayout` renders 2 contents, one for desktop and another for mobile. I plan to refactor that, but that's what we have now.